### ISSUE TITLE
FR: Needs visual numerical representations of changes while they happen.

### ISSUE THREAD

**Issue by joewintergreen** [gabrielw-us/ProBuilder-Unreal-Public-Tracker#75]
You need visual numerical representation of every change being made, and the object (or face or whatever)'s before/after dimensions. SabreCSG on Unity does this really well, as does Hammer.

SCSG examples:

[1](http://impromptugames.com/ue4/unitydoc/unitygif/sabreeasyresizeto2x.gif)
[2](http://impromptugames.com/ue4/unitydoc/unitygif/sabrebrushrotate.gif)

One thing you can see here that SCSG does wrong is visualising a cube as triangulated. Like I said in another post, it's not useful for level design to think of everything as being made out of triangles.

**Comment by gabrielw-us** [comment]
Pasting in those GIFs for simpler viewing:
![image](https://cloud.githubusercontent.com/assets/5855470/21557669/d34793fc-cdfd-11e6-941f-05bb79012d48.png)

![image](https://cloud.githubusercontent.com/assets/5855470/21557680/d9b3a92e-cdfd-11e6-9dbd-c0b78628ef53.png)

**Comment by gabrielw-us** [comment]
Agreed, this is super useful. I don't think we need to go all-out like SCSG does, but a numerical value shown would be super helpful.

Thoughts, @arthursw ?

**Comment by arthursw** [comment]
Yes, we'll do that :)

**Comment by gabrielw-us** [comment]
Adding Marketplace V1 milestone.
